__tests__/components/Header.test.js

```diff
@@ -3,16 +3,29 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 import Header from 'components/Header'
+import Config from 'Config'
 
 const props = {
   version: '1.0', // hardcode a version number for the test, in the actual app it will be set from the package.json file
 }
 
+const OLD_ENV = process.env
+
 describe('<Header />', () => {
   const wrapper = shallow(<Header.WrappedComponent {...props}/>)
 
-  it('displays the Sinopia text', () => {
-    expect(wrapper.find('h1.editor-logo').text()).toBe('LINKED DATA EDITOR')
+  beforeAll(() => {
+    process.env = {
+      SINOPIA_ENV: 'TEST',
+    }
+  })
+
+  it('sinopia env overrides static value for display', () => {
+    expect(Config.sinopiaEnv).toEqual(' - TEST')
+  })
+
+  it('displays the Sinopia text with the environment name', () => {
+    expect(wrapper.find('h1.editor-logo').text()).toBe('LINKED DATA EDITOR - TEST')
```

Comment:
There is one failing test, but I'm not sure why.  The `Config.sinopiaEnv` variable is getting setup correctly (as confirmed by the previous passing test) but is not being picked up the component correctly.  It works fine in the browser.